accountCredentials.md

```diff
@@ -17,6 +10,40 @@ Specific information for finding your credentials can be found [here](https://su
 | `subAccountId` <br> -or- <br> `siteId` | Any **id** for any sub-account.  The `subAccountId` is used as part of the url to make some API requests. <br> Ex: `https://dashboard.bandwidth.com/api/accounts/{accountId}/sites/{sideId}` <br><br> ℹ️ *Sub-accounts* are also known as *sites* <br> ℹ️ An *account* can have multiple *sub-accounts*|  `13606`                              |
-| `userId`                               | Your Bandwidth Account ID. The `userId` is used as part of the url to make API requests. This may be the same value as your accountId. <br> EX: `https://messaging.bandwidth.com/api/v2/users/{userId}/messages` <br><br> ⚠️ This **IS NOT** the same as your log in! ⚠️                                                                                |  `u-sd89g93epa2nb4euazd83kv8` OR `920012` |
+| `userId`                               | Your Bandwidth Account ID. The `userId` is used as part of the url to make API requests. This may be the same value as `accountId`. <br> EX: `https://messaging.bandwidth.com/api/v2/users/{userId}/messages` <br><br> ⚠️ This **IS NOT** the same as your log in! ⚠️                                                                                |  `u-sd89g93epa2nb4euazd83kv8` OR `920012` |
 | `apiToken`                             | Your unique api token.  The `apiToken` is used as the **username** to authenticate to the API as part of the basic auth scheme                                                                                                                                                                       |  `f12a9edeed04ecd21b303c6f1f9f0831a1482f7f3c59199e`          |
 | `apiSecret`                            | Your unique api secret.  The `apiSecret` is used as the **password** to authenticate to the API as part of the basic auth scheme                                                                                                                                                                     |  `j54935lddasl837592356aasdf8359hlo3` |
+| `applicationId` | The ID of your messaging application | `532qd-fk5odk5-dlslka40-l5k3lsdmc` |
+
+## Authentication on each API
+
+### Voice
+
+| Credential Name  | How it is used |
+|:---------------------------------------|:-------------------|
+| `accountId`                             | Used as a URL parameter for the API endpoint `https://voice.bandwidth.com/v2/account/{accountId}`   |
+| `apiToken`                             | Used as the username for HTTP basic auth  | 
+| `apiSecret`                             | Used as the password for HTTP basic auth  | 
+
+### Messaging
+
+| Credential Name  | How it is used |
+|:---------------------------------------|:-------------------|
+| `userId`                             | Used as a URL parameter for the API endpoint `https://messaging.bandwidth.com/api/v2/{userId}/messages`   |
```

Comment:
same as above, v2 world uses account id... also endpoint is `https://messaging.bandwidth.com/api/v2/users/{accountId}/messages` ya missed the `users`...

we know that `/users/accountId` is awkward. 🐢

Reply:
Bad copy paste from me 👎 

Fixed 👍